views/12-months.html

```diff
@@ -0,0 +1,39 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+You cannot use this service
+{% endblock %}
+{% block beforeContent %}
+{{ govukPhaseBanner({
```

Comment:
This phase banner is not on the other extensions templates

Reply:
Removed